core/templates/pages/collection-editor-page/collection-editor-page.directive.ts

```diff
@@ -20,13 +20,13 @@ require(
   'pages/collection-editor-page/editor-tab/collection-editor-tab.directive.ts');
 require(
   'pages/collection-editor-page/history-tab/' +
-  'collection-history-tab.directive.ts');
+  'collection-history-tab.component.ts');
```

Comment:
Can you check once, I think we don't require these "requires" anymore.

Reply:
Yup, it doesn't. Pushing